src-docs/src/views/description_list/description_list_example.js

```diff
@@ -148,14 +177,31 @@ export const DescriptionListExample = {
             <EuiCode>column</EuiCode> description lists can be presented in an
             inline, column format.
           </p>
+        </Fragment>
+      ),
+      snippet: descriptionListColumnSnippet,
+      demo: <DescriptionListColumn />,
+    },
+    {
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: descriptionListResponsiveColumnSource,
+        },
+      ],
+      text: (
+        <Fragment>
           <p>
             To return to the typical row format on smaller screens set{' '}
-            <EuiCode>type</EuiCode> to <EuiCode>responsiveColumn</EuiCode>.
+            <EuiCode>type</EuiCode> to <EuiCode>responsiveColumn</EuiCode>.{' '}
+            <em>
+              The following list will is only column format on larger screens.
+            </em>
```

Comment:
Just my my 2c, but I don't think the `<em>` tags here are super necessary - the sentence makes sense and is helpful enough without italics. No worries if you'd rather leave this as is though!

Reply:
Oh, just noticed a minor grammar nit here
```suggestion
            <em>
              The following list will only show the column format on larger screens.
            </em>
```